src/main/webapp/app/shared/utils/UrlUtils.tsx

```diff
@@ -68,31 +70,35 @@ export const MSILink: React.FunctionComponent<{}> = () => {
 export const CitationLink: React.FunctionComponent<{
   content: string;
 }> = props => {
-    const regexps = [REGEXP.PMID, REGEXP.NCTID];
-    let contentWithLink = props.content;
-    _.forEach(regexps, (regexp) => {
-      const results = _.uniq(props.content.match(new RegExp(regexp, 'ig')));
-      if (results) {
-        _.forEach(results, (matchedItem) => {
-          const datum = matchedItem;
-          switch ( regexp ) {
-            case REGEXP.PMID:
-              contentWithLink = contentWithLink.replace( new RegExp( `${datum}(?!s*,)`, 'g' ),
-                `<a href="${REGEXP_LINK[regexp]}${datum.split( ':' )[ 1 ].trim()}" target="_blank">${datum}</a>` );
-              break;
-            case REGEXP.NCTID:
-              contentWithLink = contentWithLink.replace( datum,
-                `<a href="${REGEXP_LINK[regexp]}${datum}" target="_blank">${datum}</a>` );
```

Comment:
@victoria34 we should avoid wrap element in string. It will prevent the typescript checks.